.travis.yml

```diff
@@ -131,6 +131,14 @@ script:
         echo -e "Variable HOST_CLOUD not set\n"
     fi
 
+  # Check whether this build was triggered by our cron job for security-updates
+  - echo "TRAVIS_EVENT_TYPE is:" && echo $TRAVIS_EVENT_TYPE
+  - >
+    if [ "$TRAVIS_EVENT_TYPE" = 'cron' ]; then
+        echo -e "Travis built triggered via configured cron job.\n"
+        source bin/cron_script.sh
```

Comment:
Why sourcing and not executing? Do you set env variable here?

Reply:
Maybe then it's better to call this `cron_env` or similar?

Reply:
It does set some variables, but as you can see not only that. It performs both some text parsing and editing at run-time. That's why I opted for `_script`and to make sure to add enough comments within it.

Reply:
Ok! Good as it is then!